README.md

```diff
@@ -0,0 +1 @@
+See [jFCPlib](https://github.com/Bombe/jFCPlib/tree/next)
```

Comment:
```suggestion
See [jFCPlib](https://github.com/Bombe/jFCPlib).
```
So, without linking to `next` specifically because in theory the current version is on `master`.